### ISSUE TITLE
Outputting model prediction using theano backend when using Dropout

### ISSUE THREAD

**Issue by eg-999** [fchollet/keras#2763]
Hi,

I've implemented a model where I use dropout, however when I use Y_pred = model.predict(X_test, batch_size=test_batch_size), I receive an error using the theano backend. 
I know that the error is related to the learning phase (i.e. train/test phase). I don't receive this error when using the TensorFlow backend, but for some reason, this backend is painfully slow on 1D Convolutions by a factor of x10, which is why I would prefer to use the theano backend.

I know that from the FAQ's, there is a way to switch the learning phase, but I can't figure out how to implement this on Y_pred = model.predict(X_test, batch_size=test_batch_size).
Could someone please show how this can be implemented? In future updates, it would be nice if there were a switch in model.predict() function for train/test phase such as phase='test'

Thank you,
Eren

**Comment by phdowling** [comment]
It would be helpful to see your code, and the error you are getting.

**Comment by alyato** [comment]
I also can't understand the learning phase. When i  use the predict function ,it show me an error.

**Comment by eg-999** [comment]
The errors when away when I updated my cudnn to v4. Also make sure your cuda version is 7.5.
